### PR TITLE
Ava GUI: `AboutWindow` Refactor

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
**Changes:**
- Moves `AboutWindow` to a `ContentDialog`
- Update logos and fix light theme
- Redesign layout slightly
- Add a `ViewModel` and switch `AboutWindow` to compiled bindings

Screenshot 2023-01-01 at 19 03 08

### CODE REVIEW

**Comment by AcK77** [comment]
Quick "design" review (I'll take a look to the code after):

image

- The Ryujinx's version have to be put on the top of the space behind the logo (see the modified screenshot above).
- The button `See All Contributors` have to be put at the left (see the modified screenshot above).
- There are strange spacing between textblocks (like the `Maintained By:` one).
- The last text line of the patrons list is cutted:
image
- Tooltips on link buttons are missing.
- Thog's name is "marysaka" now.

File: Ryujinx.Ava/UI/ViewModels/AboutWindowViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            set
+            {
+                _version = value;
+                OnPropertyChanged();
+            }
+        }
+
+        public string Developers
+        {
+            get => string.Format(LocaleManager.Instance[LocaleKeys.AboutPageDeveloperListMore], "gdkchan, Ac_K, Thog, rip in peri peri, LDj3SNuD, emmaus, Thealexbarney, Xpl0itR, GoffyDude, »jD«");
```
Comment (AcK77) [inline Ryujinx.Ava/UI/ViewModels/AboutWindowViewModel.cs:86]:
```suggestion
            get => string.Format(LocaleManager.Instance[LocaleKeys.AboutPageDeveloperListMore], "gdkchan, Ac_K, marysaka, rip in peri peri, LDj3SNuD, emmaus, Thealexbarney, Xpl0itR, GoffyDude, »jD«");
```

File: Ryujinx.Ava/UI/ViewModels/AboutWindowViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+            var assets = AvaloniaLocator.Current.GetService<Avalonia.Platform.IAssetLoader>();
+
+            if (ConfigurationState.Instance.Ui.BaseStyle.Value == "Light")
+            {
+                GithubLogo = new Bitmap(assets.Open(new Uri("resm:Ryujinx.Ui.Common.Resources.Logo_GitHub_Light.png?assembly=Ryujinx.Ui.Common")));
+                DiscordLogo = new Bitmap(assets.Open(new Uri("resm:Ryujinx.Ui.Common.Resources.Logo_Discord_Light.png?assembly=Ryujinx.Ui.Common")));
+                PatreonLogo = new Bitmap(assets.Open(new Uri("resm:Ryujinx.Ui.Common.Resources.Logo_Patreon_Light.png?assembly=Ryujinx.Ui.Common")));
+                TwitterLogo = new Bitmap(assets.Open(new Uri("resm:Ryujinx.Ui.Common.Resources.Logo_Twitter_Light.png?assembly=Ryujinx.Ui.Common")));
+            } else
```
Comment (AcK77) [inline Ryujinx.Ava/UI/ViewModels/AboutWindowViewModel.cs:101]:
```suggestion
            }
            else
```

File: Ryujinx.Ava/UI/Windows/AboutWindow.axaml.cs [hunk 3]
```diff
@@ -1,70 +1,55 @@
...
+            if (sender is Button button)
             {
-                Supporters = LocaleManager.Instance[LocaleKeys.ApiError];
+                OpenHelper.OpenUrl(button.Tag.ToString());
             }
-
-            await Dispatcher.UIThread.InvokeAsync(() => SupportersTextBlock.Text = Supporters);
         }
 
         private void AmiiboLabel_OnPointerPressed(object sender, PointerPressedEventArgs e)
```
Comment (AcK77) [inline Ryujinx.Ava/UI/Windows/AboutWindow.axaml.cs:55]: Not related to that part of code, but maybe we should do something to show the amiibo website link is clickable.

**Review by AcK77 — CHANGES_REQUESTED** [review]
Some changes here and there.
Maybe we should name white logos > `Logo_Dummy_White` instead of `Light`, same for `Dark` > `Black` ?

File: Ryujinx.Ava/UI/ViewModels/AboutWindowViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            set
+            {
+                _version = value;
+                OnPropertyChanged();
+            }
+        }
+
+        public string Developers
+        {
+            get => string.Format(LocaleManager.Instance[LocaleKeys.AboutPageDeveloperListMore], "gdkchan, Ac_K, marysaka, rip in peri peri, LDj3SNuD, emmaus, Thealexbarney, Xpl0itR, GoffyDude, »jD«");
```
Comment (AcK77) [inline Ryujinx.Ava/UI/ViewModels/AboutWindowViewModel.cs:86]:
```suggestion
            get => string.Format(LocaleManager.Instance[LocaleKeys.AboutPageDeveloperListMore], "gdkchan, Ac_K, marysaka, rip in peri peri, LDj3SNuD, emmaus, Thealexbarney, GoffyDude, TSRBerry, IsaacMarovitz");
```

**Review by AcK77 — APPROVED** [review]

**Review by marysaka — APPROVED** [review]

**Comment by Foul-Tarnished** [comment]
why the uppercase for "By" hmmm

**Pull request merged**